FABRIC-AND-CFT.md

```diff
@@ -4,85 +4,85 @@ This page highlights the main differences (both technical and philosophical) bet
 
 ## Cloud Foundation Fabric (a.k.a Fabric, this repo)
 
-Fabric is a collection of Terraform modules and end to end examples meant to be cloned as a single unit and used as is for fast prototyping or decomposed and modified for usage in organizations.
+Fabric is a collection of Terraform modules and end-to-end examples meant to be cloned as a single unit and used as is for fast prototyping or decomposed and modified for usage in organizations. Fabric also provides a robust implementation of foundation [stages](./fast/stages), offering a more complete blueprint for organizations to build upon.
 
 ## Cloud Foundation Toolkit (a.k.a CFT)
 
-CFT is a collection of Terraform modules and examples with opinionated GCP best practices implemented as individual modules for gradual adoption and off the shelf usage in organizations.
+CFT is a collection of Terraform modules and examples with opinionated GCP best practices implemented as individual modules for gradual adoption and off the shelf usage in organizations. CFT also includes an IaC [foundation repository](https://github.com/terraform-google-modules/terraform-example-foundation), though it's less developed compared to Fabric's.
```

Comment:
Strictly speaking Fabric and FAST are separate things. Same applies to CFT and SFB.  

This document intentionally focused only on Fabric and SFB.